mongo/datadog_checks/mongo/api.py

```diff
@@ -49,49 +36,36 @@ def server_info(self, session=None):
     def list_database_names(self, session=None):
         return self._cli.list_database_names(session)
 
-    def _initialize(self):
+    def _initialize(self, replicaset=None):
```

Comment:
what is replicaset? A boolean, an object? Maybe this is a good chance to document it in `__init__`